client/end.py

```diff
@@ -0,0 +1,28 @@
+from definitions import *
+import pygame as pg
+
+from buttonPress import Button
+
+def endScreen(screen):
+    button_rejoin = Button(90, 30, "Rejoin", (50,130))
+    back_to_login = Button(200, 30, "Return to Login Screen", (50,180))
+    
+    done = False
+    rejoin = False
+    while not done:
+        for event in pg.event.get():
+            if event.type == pg.QUIT:
+                sys.exit(0)
+            elif event.type == pg.MOUSEBUTTONDOWN:
+                if button_rejoin.collidepoint(event.pos) or back_to_login.collidepoint(event.pos):
```

Comment:
the activeness of each of the buttons are determined by if itself is clicked. it does not make sense that clicking on your your rejoin button activates the back to login button